index.html

```diff
@@ -0,0 +1,235 @@
+                    </p>
+                </div>
+            </div>
+            <div class="program p2">
+                <div class="program-image">
+                    <img src="./images/prog_icon_02.png" alt="">
+                </div>
+                <div class="program-name">
+                    <h3>Speaker</h3>
+                </div>
+                <div class="program-comment">
+                    <p>
+                        Be empowered by the words of knowledge and inspiration from our various speakers.
+                    </p>
+                </div>
+            </div>
+            <div class="program p3">
+                <div class="program-image">
+                    <img src="./images/prog_icon_03.png" alt="">
+                </div>
+                <div class="program-name">
+                    <h3>Speaker</h3>
+                </div>
+                <div class="program-comment">
+                    <p>
+                        Be empowered by the words of knowledge and inspiration from our various speakers.
+                    </p>
+                </div>
+            </div>
+            <div class="program p4">
+                <div class="program-image">
+                    <img src="./images/prog_icon_04.png" alt="">
+                </div>
+                <div class="program-name">
+                    <h3>Speaker</h3>
+                </div>
+                <div class="program-comment">
+                    <p>
+                        Be empowered by the words of knowledge and inspiration from our various speakers.
+                    </p>
+                </div>
+            </div>
+            <div class="program p5">
+                <div class="program-image">
+                    <img src="./images/prog_icon_05.png" alt="">
+                </div>
+                <div class="program-name">
+                    <h3>Speaker</h3>
+                </div>
+                <div class="program-comment">
+                    <p>
+                        Be empowered by the words of knowledge and inspiration from our various speakers.
+                    </p>
+                </div>
+            </div>
+
+        </div>
+        <div class="program-button">
+            <button>Registration for projects</button>
+        </div>
+    </section>
+
+    <section class='speakers container' id="speaker">
+        <h2>Feature Speakers</h2>
+        <hr class="line">
+
+        <div class='speaker-area'>
+        </div>
+
+    </section>
+
+    <section class="partners">
+        <h2>partners</h2>
+        <hr class="line">
+        <div class="partners-container">
+
+            <img src="./images/airbnb.svg" alt="">
+
+            <img src="./images/google.svg" alt="">
+            <img src="./images/kakao.svg" alt="">
+            <img src="./images/mozilla.svg" alt="">
+            <img src="./images/naver.svg" alt="">
+
+        </div>
+    </section>
+
+    <section class="footer">
```

Comment:
- [ ] I would like to use the footer tag instead of using the footer class for a semantic reason and because A `<footer>` typically contains information about the author of the section, copyright data, or links to related documents. :+1:

Reply:
- [ ] @kazaneza as previously suggested kindly use a semantic tag in this case `footer` .